First_Task_using_Tailwind_version_2_2_19.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <!-- Using Tailwind CSS -->
+  <link href="custom_tailwind.css" rel="stylesheet">
+  <title>Webpage created using Tailwind CSS</title>
+</head>
+
+<body class="m-0 p-0 font-sans">
+<!-- First page -->
+<!-- First 2/5th of the screen -->
+<div class="box-border flex h-screen40 relative">
+  <div class="bg-green-500 w-2/3 relative border-black border-t-2 border-r-2 border-l-2">
```

Comment:
When using flex, use style: flex-1 instead of w-2/3. Since it is not available in tailwind, You can use style and class both. Or you can add classes to tailwind.config.js.